cinema/serializers.py

```diff
@@ -1 +1,84 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Actor,
+    Genre,
+    CinemaHall,
+    Movie,
+    MovieSession
+)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ("first_name", "last_name", "full_name")
```

Comment:
2. At the same time movie detail endpoint should provide complete information about the genres and actors.
from README. We will not see actors' ids.